src/views/ResultsPage/ResultsPage.jsx

```diff
@@ -474,7 +475,10 @@ const ResultsPage = () => {
 										Print Selected
 									</button>
 								</div>
-								<div className="results-page__pager">
+								<div
+									className={`results-page__${
+										pagerExists ? `pager` : `no_pages`
```

Comment:
Did you try this?  The no_pages is now a modifier, so it would be `results-page__pager--no-pages` and no longer `results-page__pager--no-pages`

Reply:
what?

Reply:
what change do you need me to make?

Reply:
wait nevermind I think I understand what you meant